content/post/2024-03-26-my-office-is-talking-shit/index.zh.md

```
@@ -0,0 +1,377 @@
+
+负责抓取传感器数据并让 LLM 说段子的代码：  
+[harperreed/houseagent](https://github.com/harperreed/houseagent)
+
+给智能体装“眼睛”的代码：  
+[harperreed/eyeballs-mqtt](https://github.com/harperreed/eyeballs-mqtt)
+
+理论上并不难搭，但也谈不上“一键部署”。这套东西已经连续跑了 6–8 个月，几乎没怎么动过，经常把我们逗得哈哈大笑，也让来访的朋友发出“WTF”感叹。如果你动手试了，务必告诉我；有问题就 [发邮件](mailto:harper@modest.com)，也许我能帮上忙！
+
+我猜，用不了多久 Home Assistant 就能直接搞定这一切。
+
+**Part 2 稍后上线——到时再告诉大家我们如何用一套 VTuber 装备给智能体安上“身体”**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)